### PR TITLE
[Rule Tuning] Local Service Commands

### LINKED ISSUE #1000: [Rule Tuning] Local Service Commands
## Description
As is, the `Local Service Commands` rule is too noisy. It can be further refined to eliminate noise. 

## Original Rule
```
event.category:process and event.type:(start or process_started) and process.name:sc.exe and process.args:(config or create or failure or start)
```

## Potential Update

```
process where event.type == "start" and
  (process.name : "sc.exe" or process.pe.original_file_name : "sc.exe") and
  process.parent.name : ("cmd.exe", "wscript.exe", "rundll32.exe", "regsvr32.exe",
    "wmic.exe", "mshta.exe","powershell.exe", "pwsh.exe") and
  process.args:("config", "create", "start") and
  /* exclude SYSTEM SID - look for service creations by non-SYSTEM user */
  not user.id : "S-1-5-18"
```

### PR DESCRIPTION
**Pull request by bm11100**
## Issues
resolves #1000 

## Summary
As is, the `Local Service Commands` rule is too noisy. It can be further refined to eliminate noise. This PR updates the rule to look for service control commands spawning from a script interpreter process by a non SYSTEM user. 

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/lateral_movement_service_control_spawned_script_int.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and
+  (process.name : "sc.exe" or process.pe.original_file_name : "sc.exe") and
+  process.parent.name : ("cmd.exe", "wscript.exe", "rundll32.exe", "regsvr32.exe",
+                        "wmic.exe", "mshta.exe","powershell.exe", "pwsh.exe") and
```
Comment (brokensound77) [inline rules/windows/lateral_movement_service_control_spawned_script_int.toml:28]:
```suggestion
                         "wmic.exe", "mshta.exe","powershell.exe", "pwsh.exe") and
```

File: rules/windows/lateral_movement_service_control_spawned_script_int.toml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+risk_score = 21
+rule_id = "e8571d5f-bea1-46c2-9f56-998de2d3ed95"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and
+  (process.name : "sc.exe" or process.pe.original_file_name : "sc.exe") and
```
Comment (brokensound77) [inline rules/windows/lateral_movement_service_control_spawned_script_int.toml:26]:
```suggestion
  (process.name : "sc.exe" or process.pe.original_file_name == "sc.exe") and
```

File: rules/windows/lateral_movement_service_control_spawned_script_int.toml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  process.parent.name : ("cmd.exe", "wscript.exe", "rundll32.exe", "regsvr32.exe",
+                        "wmic.exe", "mshta.exe","powershell.exe", "pwsh.exe") and
+  process.args:("config", "create", "start") and
```
Comment (brokensound77) [inline rules/windows/lateral_movement_service_control_spawned_script_int.toml:29]:
```suggestion
  process.args:("config", "create", "start", "delete", "stop", "pause") and
```

**Review by brokensound77 — APPROVED** [review]
a few comments then LGTM 👍

**Review by Samirbous — APPROVED** [review]
👍

**Pull request merged**